l10n_it_ricevute_bancarie/views/configuration_view.xml

```diff
@@ -31,6 +31,11 @@
                     <separator colspan="4" string="Settlement"/>
                     <field name="settlement_journal_id" />
                 </group>
+                <group string="After Collection" attrs="{'invisible': [('type','!=','incasso')]}" colspan="4" col="4">
+                    <separator colspan="4" string="Acceptance"/>
+                    <field name="acceptance_journal_id" attrs="{'required':[('type','=','incasso')]}"/>
```

Comment:
Non è meglio non duplicare i campi nel form? Vengono salvati comunque correttamente?

Reply:
Grazie della review!
> Non è meglio non duplicare i campi nel form?

_Meglio_ in che senso? Per come è strutturato il resto del form, mi è sembrato più pulito avere una sezione a parte per la nuova configurazione; l'alternativa sarebbe stata aggiungere `attrs` in quasi ogni riga, oppure rivoluzionare non poco l'XML esistente.
Fosse stato codice Python, avrei potuto estrarre un pezzo per non ripeterlo ecc.

> Vengono salvati comunque correttamente?

Sì